aixplain/modules/agent/tool/model_tool.py

```diff
@@ -76,4 +76,11 @@ def __init__(
             model = model.id
         self.supplier = supplier
         self.model = model
-        self.function = function
\ No newline at end of file
+        self.function = function
+
+    def validate_model_tool():
+        from aixplain.factories.model_factory import ModelFactory
+        try:
+            model = ModelFactory.get(model)
+        except:
+            raise Exception("Model Tool Unavailable.")
```

Comment:
Write a more detailed message. Something like:

f"Model Tool Unavailable. Make sure model '{model}' exists or you have access to it."